[DOCS] Add warning about bypassing ML PUT APIs (#38509)
elastic/elasticsearch: docs/reference/ml/apis/put-job.asciidoc

`PUT _ml/anomaly_detectors/<job_id>`

===== Description

> IMPORTANT:  Putting a job must be done via this API only. Do not put a job

Reviewer: ```suggestion
IMPORTANT: You must use {kib} or this API to create a {ml} job. Do not put a job
```

>             directly to the `.ml-config` index using the Elasticsearch Index API.

Reviewer: ```suggestion
            directly to the `.ml-config` index using the Elasticsearch index API.
```

>             If {es} {security-features} are enabled, make sure no `write`

Reviewer: ```suggestion
            If {es} {security-features} are enabled, do not give users `write`
```

>             privileges are granted to anyone over the `.ml-config` index.

Reviewer: ```suggestion
            privileges on the `.ml-config` index.
```
